parity/cli/mod.rs

```diff
@@ -315,7 +315,7 @@ usage! {
 		["Convenience options"]
 			FLAG flag_unsafe_expose: (bool) = false, or |c: &Config| c.misc.as_ref()?.unsafe_expose,
 			"--unsafe-expose",
-			"All servers will listen on external interfaces and will be remotely accessible. It's equivalent with setting the following: --{{ws,jsonrpc,ui,ipfs,secret_store,stratum}}-interface=all --*-hosts=all    This option is UNSAFE and should be used with great care!",
+			"All servers will listen on external interfaces and will be remotely accessible. It's equivalent with setting the following: --[ws,jsonrpc,ui,ipfs-api,secretstore,stratum,dapps,secretstore-http]-interface=all --*-hosts=all    This option is UNSAFE and should be used with great care!",
```

Comment:
Well, I don't really understand the options :smile: 

ws = --ws-apis ?
jsonrpc == --jsonrpc-apis?
ui ? 
ipfs-api == --ipfs-api?
stratum == --stratum?
dapps == --dapps-apis-all?
secretstore == ?
secretstore-http == --secretstore-http-interface

Can we clarify this consistently (This might be out-of-scope for this PR but ...)?

Reply:
also had problems with understanding this one at first, but it's all right

```
ws = --ws-interface
jsonrpc == --jsonrpc-interface
ui == --ui-interface
ipfs-api == --ipfs-interface
stratum == --stratum-interface
dapps == --dapps-interface
secretstore == secretstore-interface
secretstore-http == --secretstore-http-interface
```

Reply:
Ok, then I suggest the change `ipfs-api` to `ipfs` for consistency then it's good to go IMO

Reply:
I just added the names from the available interfaces (https://github.com/paritytech/parity/blob/a002cac95a8a998686215e6b517b5e79986cbaef/parity/cli/mod.rs#L575).
Feel free to change the names and other things to make it clear, this was definitely not what I intended to do for this PR, just making sure things are consistent and up to date.